File: RIO2025_Semifinal_S1 (42).py
l, r = 8, 12
if 1 <= l <= r and r <= 10 ** 18:
    if (r - l + 1) <= 10 ** 5:
        for i in range(l, r + 1):
            c = str(i)
            if c == c[::-1]:
                print("Palindrome!")
            else:
                print(i)

File: RIO2025_Semifinal_S1 (42).cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{

    int l,r;
    cin>>l>>r;
    string s="";
    for(int i=l;i<r+1;i++){
        if(l<r){
            if(i%10==0){
                cout<<i<<endl;
        }
            else if(i/10==0){
                cout<<"Palindrome!"<<endl;
        }
            else if(i/100==i%10 && (i/10)>=10 ){
                cout<<"Palindrome!"<<endl;
            }
            else if (i%10!=0 && i%10==i/10){
                cout<<"Palindrome!"<<endl;
        }
            else if (i%11==0 || i%111==0 || i%1111==0 || i%11111==0 ||i%111111==0 || i%1111111==0 || i%11111111==0 || i%111111111==0 ){
                cout<<"Palindrome!"<<endl;
        }
        else{
            cout<<i<<endl;
        }
        }

    }


}

